src/pages/homePage/apiHomePage.ts

```diff
@@ -4,32 +4,48 @@ import { Dish } from "../../shared/interfaces/Dish";
 import CardItem from "../../shared/interfaces/CardItem";
 import { Chef, ChefFromDB } from "../../shared/interfaces/Chef";
 import { Restaurant, RestaurantFromDB } from "../../shared/interfaces/Restaurant";
-import { transform } from "typescript";
 
 const BASE_URL = process.env.REACT_APP_BASE_URL;
 
 export const fetchDishes = async (): Promise<Dish[]> => {
     try {
         const response = await axios.get<Dish[]>(`${BASE_URL}/dishes`);
-        const transformedDishes = response.data.map((dish) => ({
-            ...dish,
-            ingredientsString: dish.ingredients.join(", "),
-        }));
+        const transformedDishes = processDishes(response.data);
         return transformedDishes;
-        // return response.data;
     } catch (error) {
         console.error("Failed to fetch Dishes:", error);
         throw error;
     }
 };
 
+export const fetchSignatureDishes = async (): Promise<Dish[]> => {
+    try {
+        const response = await axios.get<Dish[]>(`${BASE_URL}/dishes/signature`);
```

Comment:
It's recommended to have the axios in a separate service that will be responsible for making the calls and returning the response. Here we are supposed to call this function but not directly do the request

Reply:
Also the try/catch will be on the service and not here, to catch errors that come from the server